examples/human_chr_selection.toml

```diff
@@ -10,7 +10,7 @@ reference = "/path/to/reference.mmi"
 name = "select_chr_21_22"
 control = false
 min_chunks = 0
-max_chunks = inf
+max_chunks = 12
```

Comment:
We need to come back to this - it is now meaningless really.

Reply:
I agree with chunk size of 12, but yes this needs clarification because chunks are different now